README.md

```diff
@@ -28,7 +28,41 @@ This will produce a `mexport.so` library which needs to be placed in the IPFS pl
 
 ## Configuration
 
-This will eventually have configuration via the usual IPFS config mechanism.
+This plugin can be configured using the usual IPFS configuration.
+
+```
+...
+"Plugins": {
+    "Plugins": {
+      "metric-export-plugin": {
+        "Config": {
+          "QueryPeerListIntervalSeconds": 10,
+          "TCPServerConfig": {
+            "ListenAddress": "localhost:8181"
```

Comment:
We have to double check that this does not expose the TCP Server API to the outside as this could open up potential vulnerabilities